composer.json

```diff
@@ -5,6 +5,7 @@
   "type": "wordpress-plugin",
   "license": "GPL-2.0+",
   "require": {
+    "php": ">=5.6",
     "composer/installers": "1.7.0"
```

Comment:
As `composer/installers` is needed only for plugin installation via composer, make sure that `vendor/composer/installers` directory is excluded from the built zip file (the one you get using `npm run gulp build` command).
You will need to modify `gulpfile.json` file and `plugin.files` array to exclude it.